ArduCopter/mode.h

```diff
@@ -1923,6 +1923,16 @@ class ModeFollow : public ModeGuided {
     bool allows_arming(AP_Arming::Method method) const override { return false; }
     bool is_autopilot() const override { return true; }
 
+    // the following method's return value is dubious and should be evaluated:
+    bool has_user_takeoff(bool must_navigate) const override { return true; }
```

Comment:
Follow mode has not be designed to include a takeoff safely yet. That will require more careful thought.